### ISSUE TITLE
A problem with glob syntax

### ISSUE THREAD

**Issue by el-leijona** [lukeed/taskr#143]
I have following tree structure of folders:

```
src/
|_ public/
   |_ first/
      |_ readme.txt
```

and a simple flyfile.js:

```
export default async function () {
  await this.start(["build"])
}

export async function build() {
  await this.start(["resources"])
}

export async function resources() {
  await this
    .source('src/public/**/*.*')
    .target('build', {depth: 1})
}
```

And everything is ok! But if the `readme.txt` don't have a file extension `.txt` (`readme`) and I rewrite `.source('src/public/**/*.*')` as `.source('src/public/**/*')` it will cause the EISDIR error:

```
[16:25:26] Starting "resources"
~~[~3~2~m~s~t~a~c~k~:~ ~~[~3~9~m~
~ ~ ~~[~3~2~m~-~ ~~[~3~9~m~E~r~r~o~r~:~ ~E~I~S~D~I~R~:~ ~i~l~l~e~g~a~l~ ~o~p~e~r~a~t~i~o~n~ ~o~n~ ~a~ ~d~i~r~e~c~t~o~r~y~,~ ~r~e~a~d~
~ ~ ~~[~3~2~m~-~ ~~[~3~9~m~ ~ ~ ~ ~a~t~ ~E~r~r~o~r~ ~(~n~a~t~i~v~e~)~
~~[~3~2~m~m~e~s~s~a~g~e~:~ ~~[~3~9~m~E~I~S~D~I~R~:~ ~i~l~l~e~g~a~l~ ~o~p~e~r~a~t~i~o~n~ ~o~n~ ~a~ ~d~i~r~e~c~t~o~r~y~,~ ~r~e~a~d~
~~[~3~2~m~e~r~r~n~o~:~ ~~[~3~9~m~ ~ ~~[~3~4~m~-~4~0~6~8~~[~3~9~m~
~~[~3~2~m~c~o~d~e~:~ ~~[~3~9~m~ ~ ~ ~E~I~S~D~I~R~
~~[~3~2~m~s~y~s~c~a~l~l~:~ ~~[~3~9~m~r~e~a~d~
[16:25:26] resources failed due to Error: EISDIR: illegal operation on a directory, read
```

How to solve this problem?

**Comment by lukeed** [comment]
For now, I'd go with the following:

``` js
this.source(['src/public/**/*.*', 'src/public/first/readme'])
```

Or, just rename your `readme` to `readme.md`, since that's more the standard.

The `EISDIR` error is fixed in Fly@1.0, which is still being tested.

**Comment by el-leijona** [comment]
I understand that I can rename it, but I'd like not to think about things like this) I look forward to v1.0.
Thank you!

**Comment by lukeed** [comment]
No worries, totally understandable. Closed this just because of the 1.0 fix :) 

No problemo
